tradable-app/src/components/CandlestickCharts/CandlestickCharts.js

```diff
@@ -0,0 +1,328 @@
+import React, { useState, useContext, useEffect } from "react";
+import ReactApexChart from "react-apexcharts";
+import Context from "../../store/Context";
+

```

Comment:
***GENERAL: for all components - please follow the pattern:
container - component
samrt - dumm
logic - ui